cumulusci/core/github.py

```diff
@@ -160,3 +160,13 @@ def find_previous_release(repo, prefix=None):
             most_recent = release
         else:
             return release
+
+
+def create_gist(github, description, files):
+    """Creates a gist with the given description an files.
+
+    github - an
+    description - str
+    files - A dict of files in the form of {filename:{'content': content},...}
+    """
+    return github.create_gist(description, files, public=False)
```

Comment:
Is there a good reason for this function to exist instead of just calling github.create_gist directly?

Reply:
I figured that keeping all references to `github3` in `core/github.py` would: (1) minimize effort needed to switch to a new github library if we ever wanted to, and (2) improve testability by only having to thoroughly test expected results from the github3 library in one module.